epsagon/wrappers/chalice.py

```diff
@@ -0,0 +1,31 @@
+"""
+Wrapper for AWS Lambda in Chalice environment.
+"""
+
+from __future__ import absolute_import
+from epsagon.wrappers import lambda_wrapper
+
+
+class ChaliceWrapper:
+    """
+    Class handles wrapping Chalice app.
+    In call we expect an invocation to come, and in getattr we allow `app.attr`
+    calls.
+    """
+    def __init__(self, app):
+        self.app = app
```

Comment:
You may want to use `_app` because it should not be changed (maybe add a property)

Reply:
👍